lvm2/pkg.yaml

```diff
@@ -99,6 +99,9 @@ steps:
     install:
       - |
         make DESTDIR=/rootfs install
+
+        # LVM activation is handled by Talos itself
+        rm -f /rootfs/usr/lib/udev/rules.d/69-dm-lvm.rules
```

Comment:
Isn't https://github.com/lvmteam/lvm2/blob/main/udev/95-dm-notify.rules.in also unnecessary? We seem not to have dmsetup in our rootfs if I'm not mistaken

Reply:
probably, but I don't want to make bigger changes, as this is going to be backported to 1.8 - so the minimal changes required